.circleci/config.yml

```diff
@@ -189,99 +156,41 @@ workflows:
           requires:
             - build
             - lint
+      # TODO: add back when unit tests are added
```

Comment:
I suppose it is somewhat a matter of opinion/no-big-deal, but I think better way is to just write another ticket that tells what is on todo list, than leaving commented out stuff in files. Ticket is harder to forget IMO.

Reply:
Yeah that's fair. Just a little lazy that's all :)